sonarcloud.yml

```
@@ -26,8 +26,10 @@ jobs:
       - name: Install 'cargo-sonar'
         run: cargo install cargo-sonar --version 0.9.0 --locked
       - name: Convert into Sonar compatible format
+        if: ${{ env.DOCKER_ORG == 'navitia' }}
```

That will help contribution indeed. However, I'm wondering if this would not be more correct to use the following condition (see list of [environment variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables)).

```suggestion
        if: ${{ env.GITHUB_REPOSITORY_OWNER == 'hove-io' }}
```

---

That seems better indeed :+1: 

While we are discussion on it I was wondering if could be a good idea to use a `DISABLE_SONARCLOUD` env variable instead, it would make it even less specific to hove-io as any fork could choose to opt-out without any change.

---

I would go the other way around. Let's make `hove-io/mimirsbrunn` have a `ENABLE_SONARCLOUD` so it's less a bother for contributors that might not know about it, what do you think? I feel the default behavior for a new contributor should just work with minimal setup.

---

Thinking about it, we might just need to test for the presence of `SONAR_TOKEN`?

---

That's a great idea, just did that :+1: